example/screens/MessageScreen.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Sample React Native App
+ *
+ * MessageScreen: Contains the selected message to be displayed.
+ */
+'use strict';
+
+import {
+ UAMessageView,
+} from 'urbanairship-react-native'
+
+import React, {
+  Component,
+} from 'react';
+
+import {
+  View,
+} from 'react-native';
+
+import styles from './../Styles'
+
+export default class MessageScreen extends React.Component {
+
+  constructor(props) {
+    super (props);
+  }
+
+  render() {
+    const { params } = this.props.navigation.state;
+    const messageId = params ? params.messageId : "";
+    return (
+       <View style={styles.backgroundContainer}>
+            <UAMessageView messageId={messageId} style={{ flex: 1 }}/>
```

Comment:
Could you add error handling and a loading indicator?

Reply:
Yes 👍